setup.py

```diff
@@ -96,6 +96,7 @@
 # 2. once modified, run: `make deps_table_update` to update src/transformers/dependency_versions_table.py
 _deps = [
     "Pillow",
+    "accelerate",
```

Comment:
```suggestion
    "accelerate>=0.7.1",
```
We need the latest release for this feature.